packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -550,16 +556,31 @@ export const ChatMessage: ComponentWithAs<'div', ChatMessageProps> &
       }),
     });
 
+    const bodyElement = Box.create(comfyBody || {}, {
+      defaultProps: () =>
+        getA11Props('comfyBody', {
+          className: chatMessageSlotClassNames.comfyBody,
+          styles: resolvedStyles.comfyBody,
+        }),
+      overrideProps: () => ({
+        content: (
+          <>
+            {actionMenuElement}
+            <div className={chatMessageSlotClassNames.bar} />
+            {badgePosition === 'start' && badgeElement}
+            {messageContent}
+            {reactionGroupPosition === 'end' && reactionGroupElement}
+            {badgePosition === 'end' && badgeElement}
+            {readStatusElement}
+          </>
+        ),
+      }),
+    });
+
     elements = (
```

Comment:
after this change, `elements` seems to be the same as `bodyElement` so we probably do not need it and we can just replace it with `bodyElement`